### PR TITLE
Do not assume a male player in code comments

### PR DESCRIPTION
**Pull request by HybridDog**
I may not have found every related comment.

### CODE REVIEW

**Comment by weluvgoatz** [comment]
Currently, the level editor defaults a level author's name to "Mr. X". You
should change this to "SuperTux Player".

**Comment by HybridDog** [comment]
> You should change this to "SuperTux Player".

Done.

**Comment by weluvgoatz** [comment]
Thank you. :)

File: src/object/camera.cpp [hunk 1]
```diff
@@ -366,7 +366,7 @@ Camera::update_scroll_normal(float dt_sec)
   }
   if (ymode == 2) {
     // target_y is the high we target our scrolling at. This is not always the
-    // high of the player, but if he is jumping upwards we should use the
+    // high of Tux, but if he is jumping upwards we should use the
```
Comment (tobbi) [inline src/object/camera.cpp]: Change "he is" to "they are" maybe?

Comment (HybridDog) [reply src/object/camera.cpp]: I've changed more things in this comment now.
I don't understand the meaning of this comment. It says that in the future there probably needs to be exceptions for trampolines although these already exist in SuperTux for a long time.
https://github.com/SuperTux/supertux/blob/7d339fe2a3001c3a8c5a71094b2b3deac7bb4d52/src/object/player.hpp#L43

**Pull request merged**